config/config.go

```diff
@@ -478,18 +498,33 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 		return fmt.Errorf("root route must not have any mute time intervals")
 	}
 
+	if len(c.Route.ActiveTimeIntervals) > 0 {
+		return fmt.Errorf("root route must not have any active time intervals")
+	}
+
 	// Validate that all receivers used in the routing tree are defined.
 	if err := checkReceiver(c.Route, names); err != nil {
 		return err
 	}
 
 	tiNames := make(map[string]struct{})
-	for _, mt := range c.MuteTimeIntervals {
-		if _, ok := tiNames[mt.Name]; ok {
-			return fmt.Errorf("mute time interval %q is not unique", mt.Name)
+	// read mute time intervals until deprecated
+	if len(c.MuteTimeIntervals) > 0 {
```

Comment:
We should probably consider that `mute_time_intervals` and `time_intervals` can coexist and that `mute_time_intervals` can be used indifferently for mute and active intervals.
In which case, I would expect that we check name uniqueness across both slices.

Reply:
I see I will check this, thanks